Sources/Nats/NatsConnection.swift

```diff
@@ -351,6 +351,35 @@ class ConnectionHandler: ChannelInboundHandler {
             initialConnect.signature = base64sig
             initialConnect.userJwt = String(data: jwt, encoding: .utf8)!
         }
+        if let nkey = self.auth?.nkeyPath {
```

Comment:
should we check if both `nkey` and `nkeyPath` are set and throw if it's the case?